create_shared_media_report.py

```diff
@@ -0,0 +1,103 @@
+import requests
+import argparse
+from datetime import datetime
+import json
+import sys
+
+class SharedMediaReport:
+    #Set up workspace for API calls to Rhombus Systems
+    def __init__(self,args):
+        #Initialize argument parser
+        self.argParser = self.__initArgParser()
+        self.args = self.argParser.parse_args(args)
+        #Create a session to set default call Header information
+        self.session = requests.session()
+        self.session.headers = {"x-auth-scheme": "api-token","x-auth-apikey": self.args.apiKey}
+        #Set a default base URL for api calls
+        self.url = "https://api2.rhombussystems.com/api/"
+        self.curTime = int(datetime.utcnow().timestamp())
+
+    #Define arguments which the user may be prompted for
+    @staticmethod
+    def __initArgParser():
+        argParser = argparse.ArgumentParser(description='Generates a simple report on the status of shared media within the organization')
+        argParser.add_argument("apiKey", help = 'Your personal api key, get this from your Rhombus Console')
+        return argParser
+    
+    def execute(self):
+        #Collect Data and ensure expected status codes are received
+        response = self.session.post(self.url+"event/getSharedClipGroupsV2")
+        if response.status_code != 200:
+            print("Encountered an Error")
+            return
+        sharedClipGroups = json.loads(response.text)
+        sharedClipGroups = sharedClipGroups["sharedClipGroups"]
+
+        response = self.session.post(self.url+"video/getSharedTimelapseGroups")
+        if response.status_code != 200:
+            print("Encountered an Error")
+            return
+        sharedTimelapseGroups = json.loads(response.text)
+        sharedTimelapseGroups = sharedTimelapseGroups["sharedTimelapses"]
+
+        #Camera UUIDs are needed to check for open streams
+        response = self.session.post(self.url+"camera/getMinimalList")
```

Comment:
`/camera/getMinimalList` is deprecated.  Use `/api/camera/getMinimalCameraStateList` instead